From a0485b9f99c5577ba4e56cf90b22be03f50f3674 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Fri, 22 Dec 2017 16:06:34 +0100
Subject: [PATCH 19/19] sepolicy: Allow energyawareness to read sysfs files

* Old blobs don't have energyawareness specific sysfs
  thus we have to allow it to read any sysfs like CAF
  did in <= 7.0.

Change-Id: I80695d3d952d46ebc90cbe9497dce8b1862dc4fb
---
 legacy-common/energyawareness.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 legacy-common/energyawareness.te

diff --git a/legacy-common/energyawareness.te b/legacy-common/energyawareness.te
new file mode 100644
index 0000000..5293ecc
--- /dev/null
+++ b/legacy-common/energyawareness.te
@@ -0,0 +1 @@
+r_dir_file(energyawareness, sysfs)
-- 
2.7.4

